src/main/resources/templates/webshop/fragments/header.html

```diff
@@ -3,7 +3,7 @@
 <head></head>
 <body>
 <header class="page-header" th:fragment="header (title, image)">
-
+    <!--/*@thymesVar id="rightLevel" type="java.lang.Integer"*/-->
```

Comment:
Please name this authLevel, as it is named in the 'backend'.

Reply:
Done!